### PR TITLE
style updates for market research pages

### PR DESCRIPTION
**Pull request by schalkneethling**
Fixes based on Google doc and our meeting conversation earlier today.

- [x] This is missing the previous button functionality
- [x] Center aligning the radio buttons across browsers
- [x] Aligning the buttons to the right in the survey
- [x] Prose that replaces the deep dives feature component

I will address all of the above as soon as I have my Kuma backend up and running.

### CODE REVIEW

File: client/src/plus/deep-dives/series-card/index.scss [hunk 1]
```diff
@@ -75,3 +80,33 @@
     }
   }
 }
+
+.take-survey {
+  display: none;
+
+  @media #{layout.$mq-large-desktop-and-up} {
+    display: block;
```
Comment (dalexsy) [inline client/src/plus/deep-dives/series-card/index.scss:88]: Add `margin-bottom: 0px;`

File: client/src/plus/deep-dives/series-card/index.scss [hunk 2]
```diff
@@ -75,3 +80,33 @@
...
+  }
+
+  a.take-survey-link {
+    &:link,
+    &:visited {
+      background: linear-gradient(to right, #8565ff, #0dcbff);
```
Comment (dalexsy) [inline client/src/plus/deep-dives/series-card/index.scss:94]: Change to `linear-gradient(to right, #8565ff, #6581FF)`

File: client/src/plus/deep-dives/index.scss [hunk 3]
```diff
@@ -165,10 +181,39 @@
...
+  z-index: 99999;
+
+  @media #{layout.$mq-large-desktop-and-up} {
+    display: none;
+  }
+
+  a.take-survey-link {
+    &:link,
+    &:visited {
+      background: linear-gradient(to right, #8565ff, #0dcbff);
```
Comment (dalexsy) [inline client/src/plus/deep-dives/index.scss:202]: Change to `linear-gradient(to right, #8565ff, #6581FF)`

File: client/src/plus/deep-dives/article.tsx [hunk 4]
```diff
@@ -94,7 +93,6 @@ export default function Article({ slug }: { slug: string }) {
           />
         </div>
       </div>
-      <Feature />
       <Survey slug={slug} />
       <div
         className={`deep-dive-article-footer ${
```
Comment (dalexsy) [inline client/src/plus/deep-dives/article.tsx:98]: Sorry, I don't know where to add this, can you increase the padding for `.deep-dive-article-footer a {
    display: block;
    padding: 48px 24px;
}` like this for desktop

File: client/src/plus/deep-dives/byline/index.scss [hunk 5]
```diff
@@ -18,8 +18,14 @@
...
   }
 
-  .author-description {
-    color: theme.$mdn-neutral700;
-    margin-bottom: 0;
+  .author-links {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
```
Comment (dalexsy) [inline client/src/plus/deep-dives/byline/index.scss:25]: Add `a {color: #42435a;}`

File: client/src/plus/deep-dives/overview/app.tsx [hunk 6]
```diff
@@ -14,9 +14,20 @@ export default function App() {
...
-            <p>
-              Read a sneek peak of Rachel Andrew’s forthcoming deep dive series
-            </p>
+            <p className="author">Rachel Andrew, CSS WG Invited Expert</p>
+            <ul className="author-links">
+              <li>
+                <a href="https://twitter.com/rachelandrew" rel="external">
+                  @rachelandrew
+                </a>
+                ,
```
Comment (dalexsy) [inline client/src/plus/deep-dives/overview/app.tsx:23]: This comma doesn't show up in the rendered version

File: client/src/plus/deep-dives/index.scss [hunk 7]
```diff
@@ -165,10 +181,39 @@
...
+      color: theme.$white;
+      display: block;
+      font-size: 14px;
```
Comment (dalexsy) [inline client/src/plus/deep-dives/index.scss:205]: change to `font-size: 18px`

File: client/src/plus/deep-dives/index.scss [hunk 8]
```diff
@@ -165,10 +181,39 @@
...
+      font-weight: bold;
+      margin-top: layout.$base-spacing;
+      min-height: 40px;
+      padding: math.div(layout.$base-spacing, 2);
```
Comment (dalexsy) [inline client/src/plus/deep-dives/index.scss:209]: change to `padding: 20px;`

**Review by dalexsy — COMMENTED** [review]
Reviewed styling changes

File: client/src/plus/deep-dives/overview/app.tsx (hunk 6 above)
Comment (schalkneethling) [reply client/src/plus/deep-dives/overview/app.tsx:23]: Screenshot 2021-07-28 at 15 20 27

It shows up for me. Unless you are referring to something else?

Comment (dalexsy) [reply client/src/plus/deep-dives/overview/app.tsx:23]: oh, sorry, must have taken the wrong one, I meant the in article links 
image

**Pull request merged**